src/main/scala/chisel3/tester/package.scala

```diff
@@ -174,4 +183,11 @@ package object tester {
   implicit def decoupledToDriver[T <: Data](x: ReadyValidIO[T]) = new DecoupledDriver(x)
 
   implicit def validToDriver[T <: Data](x: ValidIO[T]) = new ValidDriver(x)
+
+  def intervalToBigDecimal(i: Interval): BigDecimal = {
```

Comment:
Yes, it should, but doing so can have API implications that need to be discussed and it's not a small change if done right because if should be done symmetrically for all types, FixedPoint has a potential bug related to this. See Chisel Issue #1272.
But in the meantime it would be nice to have Interval working in here.

Reply:
So what's the long-term plan for this? Once the chisel3 proper fix comes out, this goes away (unless you have an argument for two ways to do the same thing?). Which means unless it's API compatible with the planned chisel3 feature (which might cause compile-time aliasing issues?) it's going to break everyone's code.

If you want to use it as an experimental or internal feature, sure, but I don't want to give the illusion of long term support for a construct that is almost certainly disappearing in the near future. Either because we implemented it in chisel3 and it's now redundant, or for whatever reason we decided to not do it in chisel3 and those same reasons probably apply here too.